code_of_conduct.md

```diff
@@ -0,0 +1,34 @@
+# Code of Conduct
+
+All attendees, speakers, sponsors and volunteers at Practical Inclusivity SF are required to agree with the following code of conduct. Organisers will enforce this code throughout the event. We expect cooperation from all participants to help ensure a safe environment for everybody.
+
+## Need Help?
+
+If someone makes you or someone else uncomfortable at a Practical inclusivity SF event, please let an organizer know. You may find an organizer in person at the event, DM an organizer on twitter, or [fill out this form](https://nebrius.typeform.com/to/hDpXZl) and it will be sent to all of the event organizers.
```

Comment:
Note: I created the TypeForm and just left it on my personal account for now. I think we can assign a custom domain to it though, which we'll probably want to do later once we have a website, right?

Reply:
Yes, I think this is the right thing to do for now